unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWActionsShould.cs

```diff
@@ -0,0 +1,190 @@
+using DCL;
+using DCL.Helpers;
+using NUnit.Framework;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+public class BIWActionsShould : IntegrationTestSuite_Legacy
```

Comment:
Can we try to use the non-legacy `IntegrationTestSuite` here? We should avoid creating new tests with the legacy class. If this causes some issues we can do some pair programming here.

Reply:
Sure, I tried with the new system but had a couple of problems, since you were on PTO I just swapped to legacy but we ca go for the new ones!